frontend/src/store/index.js

```diff
@@ -171,7 +171,10 @@ export function createStore() {
           api.getThreads(eventID).then(threads => {
             store.commit('setMessages', {});
             threads.forEach(thread => {
+              console.log('wat');
```

Comment:
Do we still need these console logs? Please remove if we don't :)

Reply:
Oops. *blush*